source/_includes/puppet-collections/_pkgutil_command_output.md

```diff
@@ -0,0 +1,11 @@
+    Package "puppet-agent-<AGENT-VERSION>-1-installer.pkg":
+       Status: signed by a certificate trusted by Mac OS X
+       Certificate Chain:
+        1. Developer ID Installer: PUPPET LABS, INC. (VKGLGN2B6Y)
```

Comment:
@shrug or @geoffnichols can you verify no changes have been made to our osx signing cert?